### DISCUSSION TITLE
The has modifier

### DISCUSSION THREAD

**Discussion by TefDesign** [tailwindlabs/tailwindcss#7968] [Help] [answered]
Hello,
I'm trying to find on Tailwind the equivalent of :has(my_class) in CSS.
Does it exist ? If not, is there an equivalent solution or do I have to use CSS directly for this ?

Thanks :)

**Comment by CFarhad** [comment, marked as answer]
hi,
Tailwindcss does not have that, but if you want to create that you can use postcss and postcss-nesting.
Its normally works but I suggest postcss.


```css
/* Normal */
.btn {
 @apply py-2 px-4 text-white rounded;
}

.btn.btn-indigo{
 @apply bg-indigo-600;
}
```

```css
/* Postcss-nesting */
.btn {
 @apply py-2 px-4 text-white rounded;

 &.btn-indigo{
  @apply bg-indigo-600;
 }
}

```

**Reply by TefDesign** [reply]
Hi @CFarhad thx :)
Indeed, this is what I did on my side. I was just wondering if it was available but I couldn't find anything in the doc. Maybe for a future evolution ;)

Thx !

**Comment by sntran** [comment]
I was thinking the same, and when [arbitrary selector](https://github.com/tailwindlabs/tailwindcss/discussions/6630#discussioncomment-2380610) is supported, then it would be possible to write plugin for this `has:`
